README.md

```diff
@@ -99,8 +104,8 @@ Clean up by executing the following commands once you are done.
 ## List of Commands
 
 
-- apkctl install platform
+- apkctl install platform --profile \<Name of Profile: cp or dp\> -n \<Namespace\> --helm-version \<Helm Version\>
```

Comment:
Why we have introduced a separate flag for this? Instead, we can identify the helm version in the code level